.pre-commit-config.yaml

```
@@ -16,3 +16,7 @@ repos:
     rev: 20.8b1
     hooks:
     -   id: black
+-   repo: https://gitlab.com/pycqa/flake8
+    rev: 3.8.4
+    hooks:
+    -   id: flake8
```

I feel like we could ignore `pycodestyle`'s errors since they can conflict with the other hooks. What do you think?

```suggestion
    -   id: flake8
        args: ['--ignore=E, W']
```

---

They are not conflicting with any other hook. I also think we should not ignore everything. I will just add a change to accept longer lines.